Build is failing with JDK 12 due to Javadoc plugin issues

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.464 s
[INFO] Finished at: 2019-07-29T12:58:23+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project microprofile-metrics-api: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR] 
[ERROR] Command line was: /opt/java/jdk-12.0.1/bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/jmartisk/Workspace/microprofile-metrics/api/target/apidocs' dir.
```

--- a/pom.xml
+++ b/pom.xml
@@ -148,6 +148,9 @@
                  <groupId>org.apache.maven.plugins</groupId>
                  <artifactId>maven-javadoc-plugin</artifactId>
                  <version>2.10.4</version>
+                 <configuration>
+                     <source>8</source>
+                 </configuration>
             </plugin>
         </plugins>
         </pluginManagement>

